src/libs/antares/study/parameters.cpp

```diff
@@ -1020,24 +925,57 @@ bool Parameters::loadFromINI(const IniFile& ini, uint version, const StudyLoadOp
     reset();
     // A temporary buffer, used for the values in lowercase
     String value;
-
+    String sectionName;
+    typedef bool (*Callback)(Parameters&, const String&, const String&, const String&, uint);
```

Comment:
For clarity, I suggest you comment each parameter
Example :
```cpp
typedef bool (*Callback)(Parameters& // [out] Parameter object to load the data into
                         const String& // [in] Key, comes left to the '=' sign in the .ini file
                         const String& // [in] Lowercase value, comes right to the '=' sign in the .ini file
                                       // etc.
                        );

Reply:
Ok : 0b051f3